### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by hemarina**

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]
I think next release might be the 0.2.0.  But we can update it later

**Comment by ellismg** [comment]
> I think next release might be the 0.2.0. But we can update it later

If we aren't 100% sure, let's keep it as `0.1.0-beta.6` for now.  We can always go change it to `0.2.0-beta.1` when the terraform stuff lands, but if we change it now going back will be confusing.

File: cli/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-0.1.0-beta.5
+0.2.0-beta.1
```
Comment (ellismg) [inline cli/version.txt:1]: I would keep this at `0.1.0-beta.6` until we are certain we are going to rev the minor version number (e.g. after the terraform or azdo changes land)

**Review by ellismg — CHANGES_REQUESTED** [review]
We should stay on 0.1.0-beta.6 for now - we can update it once we land some of the more impressive features in the next release.

**Review by ellismg — APPROVED** [review]
LGTM

**Pull request merged**